parlai/core/torch_agent.py

```diff
@@ -606,6 +612,13 @@ def add_cmdline_args(cls, argparser):
             default='\n',
             help='Join history lines with this token, defaults to newline',
         )
+        agent.add_argument(
+            '--history-add-global-end-token',
```

Comment:
Just a suggestion/comment maybe consider a short version for this arg? I just think the name is too long isn't? anyways, great work @dexterju lgtm!

Reply:
Yep that is a good idea!

Reply:
make sure it's hidden, this option shouldn't be used too much i think.